critiquebrainz/frontend/static/scripts/rating.js

```diff
@@ -0,0 +1,157 @@
+(function ($) {
```

Comment:
Original source - https://github.com/javiertoledo/bootstrap-rating-input
This plugin suited our needs the best with a few modifications. Can you please tell why are you skeptical of this kinda thing. And yes I will add the copyright.